prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div><a href="file:///C:/Users/denis/repos/prep-0421-code-solutions/prep-html-links/index.html">Click here to go to the home page </a></div>
```

Comment:
You should use a relative path here instead of an absolute path to the file on your computer.

Reply:
This has been fixed

Reply:
![Completed Updated HTML Links Page](https://user-images.githubusercontent.com/82246390/115662117-5ca4f800-a2f3-11eb-9181-e530a2dbe5a4.png)